### ISSUE TITLE
Has no method findFaster using 'this' keyword

### ISSUE THREAD

**Issue by caio-ribeiro-pereira** [arunoda/meteor-find-faster#2]
Hi, when I try to use the findFaster function it shows the error:

``` bash
Exception from sub q2dRsb4uHCmmz96nd TypeError: Object [object Object] has no method 'findFaster'
```

This is a simple function which I'm using it:

``` javascript
Post = new Meteor.Collection('posts');

Post.list = function(userIds) {
    return this.findFaster(
        {userId: {"$in": userIds}},
        {sort: {time: -1, name: 1}}
    );
};
```

**Comment by arunoda** [comment]
You can't use `this` like this. It needs to be `Post.findFaster` not `this.findFaster`

**Comment by caio-ribeiro-pereira** [comment]
ok thanks! but is there some reason why cannot use `this`?

**Comment by arunoda** [comment]
Hey I'm sorry, I didn't look at the code a closer look. Your code is correct. May be `find-faster` does not get added to the app.

try doing `meteor add find-faster`

**Comment by TimBroddin** [comment]
I'm having the same issue.

**Comment by TimBroddin** [comment]
I'm trying to do this in my (iron-)router.js file. Can you use findFaster and findOneFaster on both client and server?

**Comment by arunoda** [comment]
findFaster in server only. I've not implement methods on the client and it
has no effect. But I should implement methods on the client which simply
map the find.

---

Arunoda Susiripala

@arunoda http://twitter.com/arunoda
http://gplus.to/arunodahttps://github.com/arunoda
http://www.linkedin.com/in/arunoda

**Comment by arunoda** [comment]
I've added support for find-faster to the client side. But it will do nothing special.

**Comment by caio-ribeiro-pereira** [comment]
Thanks! Now its working using `this.findFaster` :D

**Comment by arunoda** [comment]
Cool.

On Tuesday, May 6, 2014, Caio Ribeiro Pereira notifications@github.com
wrote:

> Closed #2 https://github.com/arunoda/meteor-find-faster/issues/2.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/arunoda/meteor-find-faster/issues/2#event-118193340
> .

## 

---

Arunoda Susiripala

@arunoda http://twitter.com/arunoda
http://gplus.to/arunodahttps://github.com/arunoda
http://www.linkedin.com/in/arunoda

**Comment by TimBroddin** [comment]
Thanks a lot!
